docs/user/masternodes/setup.rst

```diff
@@ -344,7 +344,7 @@ is working. Then connect your Trezor device and click the next button
 **Test Hardware Wallet Connection** to verify the Trezor connection is
 working.
 
-.. image:: img/setup-collateral-connection.png
+.. figure:: img/setup-collateral-connection.png
```

Comment:
This fix is good. You had a second update for this file in the other PR too that got missed in this one. See: https://github.com/dashpay/docs/pull/206/files#diff-5ef9bcae9cbe3a37434dcee48274a8490ab825af04c4e7a48fd88d59e96e291f